Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, ceUniqueID):
     self.remoteScheddOptions = ""
 
   #############################################################################
-  def __writeSub(self, executable, nJobs):
+  def __writeSub(self, executable, nJobs, processors):
```

Comment:
```suggestion
  def __writeSub(self, executable, nJobs, processors=1):
```

Reply:
There shouldn't be a need for defaults in private function calls, nJobs doesn't have one either and the default is set in submitJob.